pkgs/applications/audio/sfizz/default.nix

```diff
@@ -37,18 +35,18 @@ stdenv.mkDerivation rec {
     glib
     gnome.zenity
     freetype
+    pango
   ];
   nativeBuildInputs = [ cmake pkg-config ];
 
   postPatch = ''
-  substituteInPlace editor/external/vstgui4/vstgui/lib/platform/linux/x11fileselector.cpp \
-    --replace '"/usr/bin/zenity' '"${gnome.zenity}/bin/zenity'
+    substituteInPlace plugins/editor/external/vstgui4/vstgui/lib/platform/linux/x11fileselector.cpp \
```

Comment:
> I've also ried to update, but ran into [sfztools/sfizz#505 (comment)](https://github.com/sfztools/sfizz/issues/505#issuecomment-988868693). This PR has the same issue: when zenity is not in the path, the plugin isn't able to call it, even with the patching.

I had zenity available system wide which is why I didn't notice the problem. I think I've somehow fixed the issue with the file dialog not showing but the fix on this line doesn't seem correct.

Reply:
I still don't get the dialog, unless I have zenity system wide.

> but the fix on this line doesn't seem correct.

Not sure what you mean by that.

Reply:
Did you try this branch? I don't have zenity available system wide anymore and the change I made on line 43-44 seems to have resolved it for me.

Reply:
I'm sorry; I thought I had tried the correct binary, built from this branch, but it seems like I didn't.
When I tried again just now, everything works.